lib/utils/job-utils/command-parser.js

```diff
@@ -1246,6 +1247,74 @@ function commandParserFactory(Logger, Promise, _) {
         }));
     };
 
+    CommandParser.prototype[lsfileinfo] = function(data){
+        if(data.error){
+            return Promise.resolve({ source: 'lsfileinfo', error: data.error });
+        }
+        try {
+            var lines = data.stdout.split('\n');
+            lines.splice(0, 1);
+            var parsed = [];
+            _.forEach(lines, function(line){
+                if(!line){
+                    return;
+                }
+                var properties = line.split(/\s+/);
+                if(properties[8] === '.' || properties[8] === './' || properties[8] === '..' || properties[8] === '../'){
+                    return;
+                }
+                var item = {};
+                item.link_count = properties[1];
+                item.owner = properties[2];
+                item.group = properties[3];
+                item.size = properties[4];
+                item.modifed_time = properties[5] + ' ' + properties[6] + ' ' + properties[7];
+                item.name = properties[8];
+                var permissions = properties[0];
+                var fileType = permissions.substring(0, 1);
+                var userPermissions = permissions.substring(1, 4);
+                var groupPermissions = permissions.substring(4, 7);
+                var otherPermissions = permissions.substring(7, 10);
+                item.permission = {
+                    user: userPermissions,
+                    group: groupPermissions,
+                    other: otherPermissions
```

Comment:
maybe we could simply write it as
```
user: permissions.substring(1, 4);
```
to avoid temporary variables. Only personal preference.

Reply:
Thank you Lyne! I have modified the code according to your suggestion. Appreciate for your further review. 

Besides, for the catalog result, i am not sure if it is too long to place into the description. I also attached a screenshot into the Jira.